app/tests/unit/mutations/application_review_step/updateApplicationReviewStepMutation.test.ts

```diff
@@ -0,0 +1,70 @@
+import fs from 'fs';
+import path from 'path';
+import EasyGraphQLTester from 'easygraphql-tester';
+
+const schemaCode = fs.readFileSync(
+  path.join(__dirname, '../../../../server', 'schema.graphql'),
+  'utf8'
+);
+
+const mutation = `
+        mutation updateApplicationReviewStepMutation(
+          $input: UpdateApplicationReviewStepInput!
+        ) {
+          updateApplicationReviewStep(input: $input) {
+            id
+            isComplete
```

Comment:
Instead of copying the mutation (and not actually testing the application code), you can import it. It doesn't seem to be well documented, or typed with typescript, but this works:
```ts
import {mutation} from 'mutations/application_review_step/updateApplicationReviewStepMutation';
const mutationText = (mutation as any).default.params.text;
```

and then you use `mutationText` in the tests below

Reply:
Of course this requires exporting the `mutation` variable in `mutations/application_review_step/updateApplicationReviewStepMutation`

Reply:
> Instead of copying the mutation (and not actually testing the application code), you can import it.

That does seem unwise, doesn't it! I'll admit I based the test on the one for another mutation since I wasn't too sure how to go about it - thanks for the tip, that sounds like a much better idea 😄 

We should probably also update our other mutation tests to do the same.